db/seeds.rb

```diff
@@ -4,121 +4,40 @@
 # Parallel Tests is seeding the database, so I am skipping this in the test environment
 return if Rails.env.test?
 
-def seed_courses!
-  CourseService::DefinitionLoader.call
-end
-
-def seed_lead_providers!
-  LeadProviders::Updater.call
-end
-
-def seed_itt_providers!
-  file_name = "lib/approved_itt_providers/24-11-2022/approved_itt_providers.csv"
-  ApprovedIttProviders::Update.call(file_name:)
-end
+PaperTrail.enabled = false
 
-def seed_schools
-  CSV.read(Rails.root.join("db/seeds/schools.csv"), headers: true).tap do |data|
-    if Rails.env.in?(%w[review development])
-      Rails.logger.info("Importing 1000 schools")
+Faker::Config.locale = "en-GB"
 
-      School.insert_all(data.first(1000).map(&:to_h))
-    else
-      Rails.logger.info("Importing #{data.length} schools")
+def load_base_file(file)
+  base_file = Rails.root.join("db", "seeds", "base", file)
 
-      School.insert_all(data.map(&:to_h))
-    end
-  end
+  load(base_file)
 end
 
-def seed_childcare_providers!
-  CSV.read(Rails.root.join("db/seeds/private_childcare_providers.csv"), headers: true).tap do |data|
-    if Rails.env.in?(%w[review development])
-      Rails.logger.info("Importing 1000 private childcare providers")
-
-      PrivateChildcareProvider.insert_all(data.first(1000).map(&:to_h))
-    else
-      Rails.logger.info("Importing #{data.length} private childcare providers")
-
-      PrivateChildcareProvider.insert_all(data.map(&:to_h))
-    end
+Rails.logger.info("Seeding database")
+
+seed_files = [
+  "add_cohorts.rb",
+  "add_lead_providers.rb",
+  "add_itt_providers.rb",
+  "add_courses.rb",
+  "add_schools.rb",
+  "add_childcare_providers.rb",
+  "add_statements.rb",
+  "add_settings.rb",
+].tap do |files|
+  if Rails.env.in?(%w[development review])
+    files.push(
+      "add_users.rb",
+      "add_applications.rb",
+      "add_api_tokens.rb",
+    )
   end
 end
 
-# IDs have been hard coded to be the same across all envs
-seed_childcare_providers!
```

Comment:
could we maintain the order of the seeds created here above? apart from the new required seeds?